src/sql/parts/notebook/notebook.component.ts

```diff
@@ -254,8 +262,14 @@ export class NotebookComponent extends AngularDisposable implements OnInit {
 
 		let saveNotebookButton = this.instantiationService.createInstance(SaveNotebookAction, 'notebook.SaveNotebook', localize('save', 'Save'), 'notebook-button icon-save');
 
+		const notebookBarMenu = this.menuService.createMenu(MenuId.NotebookToolbar, this.contextKeyService);
+		let groups = notebookBarMenu.getActions({ arg: null, shouldForwardArgs: true });
+		let primary: IAction[] = [];
+		let secondary: IAction[] = [];
+		fillInActions(groups, {primary, secondary}, false, (group: string) => group === 'horizontal');
```

Comment:
OK based on our chat let's not filter to specific group. In the future, we can use smart grouping for deciding what to put in dropdowns vs. showing as separate actions. the toolbar vs cell should be handled as separate menus you can choose to contribute to